modules/cnf-assigning-a-sriov-network-to-a-vrf.adoc

```diff
@@ -0,0 +1,81 @@
+[id="cnf-assigning-a-sriov-network-to-a-vrf_{context}"]
+= Assigning an SR-IOV network to a VRF
+
+[IMPORTANT]
+====
+The VRF feature described here is for Tech Preview purposes only and is not supported by Red Hat at this time.
+====
+
+As a cluster administrator, you can assign an SR-IOV network interface to your VRF domain by using the VRF CNI plug-in.
+
+To do this, you must add the VRF configuration to the optional `metaPlugins` parameter of the `SriovNetwork` resource.
+
+[id="cnf-creating-an-additional-sriov-network-with-vrf-plug-in_{context}"]
+== Creating an additional SR-IOV network attachment with the VRF CNI plug-in
+
+The SR-IOV Network Operator manages additional network definitions. When you specify an additional SR-IOV network to create, the SR-IOV Network Operator creates the NetworkAttachmentDefinition custom resource (CR) automatically.
+
+[NOTE]
+====
+Do not edit `NetworkAttachmentDefinition` CRs that the SR-IOV Network Operator manages. Doing so might disrupt network traffic on your additional network.
+====
+
+.Prerequisites
+* Install the {product-title} CLI (oc).
+* Log in to the {product-title} cluster as a user with cluster-admin privileges.
+
+.Procedure
```

Comment:
Wrong command.
Here we need to create a new resource, not edit the existing cluster one.
You can refer to it as a yaml file, as is done here https://cnf-846--ocpdocs.netlify.app/openshift-enterprise/latest/networking/hardware_networks/configuring-sriov-device.html#nw-sriov-networknodepolicy-object_configuring-sriov-device

Reply:
Ok, corrected this, submitted in latest commit.